test/core/config_test.py

```diff
@@ -144,6 +144,7 @@ def test__config__load_toml():
             "testing_bar": 7.698,
             "testing_bool": False,
             "testing_arr": ["a", "b", "c"],
+            "rules": ["L010", "L035"],
```

Comment:
Can you update these to the new codes?

Reply:
@WittierDinosaur I have updated the codes to `LT03` and `LT09`. I believe this is what you meant but please clarify if this is not the correct change.